packages/web-components/vite.config.ts

```diff
@@ -0,0 +1,62 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import vue from '@vitejs/plugin-vue';
+import pkg from './package.json' assert { type: 'json' };
+
+export default defineConfig({
+  plugins: [
+    vue({ isProduction: true, customElement: true }),
+    dts({
+      tsconfigPath: 'tsconfig.build.json'
+    }),
+    nodeResolve()
```

Comment:
```suggestion
```
You aren't bundling local packages into your package so you can remove this one too